src/Lachain.Core/Consensus/ConsensusManager.cs

```diff
@@ -326,5 +326,10 @@ public void Terminate()
         {
             _terminated = true;
         }
+
+        public EraBroadcaster GetEras(long era)
```

Comment:
I think it is a misleading name. Use GetEraBroadcaster  or smth like this please